### ISSUE TITLE
parameter :p_nls_calendar is not defined in parameter list 

### ISSUE THREAD

**Issue by MuhammadTalhaRao** [sijms/go-ora#342]
i am facing this issue when i use v2 library, i cannot use v1 because it gives me EOF and this issue has already been discussed and proposed solution for that was to move to v2

after debugging for a while i have realized that in function 
*useNamedParameters* their is following condition
*if par.Name == names[x]*

and par.Name contains values without colon like *p_nls_calendar* but on the other size values are like *:p_nls_calendar* i.e with colon which causes this issue

This function is called to deal with *GetNLS* i have executed the NLS query in my oracle db and it works fine.
 
This issue occurs when i try to execute the query, i am attaching my code below

Import 

```
go_ora "github.com/sijms/go-ora/v2"
```

To establish connection

```
dbURI := go_ora.BuildUrl(oracleConf.Addr, oracleConf.Port, oracleConf.Service, oracleConf.User, oracleConf.Password, urlOptions)

	conn, err := sql.Open(oracleConf.Driver, dbURI)
	if err != nil {
		log.Println("Can't create new oracle connection", err)
		return DB, err
	}
```

to execute the query

```
stmt, err := oracledb.Conn.Prepare(QUERY_TO_GET_DATA)
	if err != nil {
		return data, err
	}

	rows, err := stmt.Query()

	for rows.Next() {
		err = rows.Scan(&data)
		if err != nil {
			return data, err
		}
	}

	defer func() {
		err = stmt.Close()
		if err != nil {
			fmt.Println("Can't close statement: ", err)
		}

		err = rows.Close()
		if err != nil {
			fmt.Println("Can't close dataset: ", err)
		}
	}()
	
```

I was planning to move from godror to go_ora but i cannot because of this issue kindly help me with this. I am also attaching the trace log file

**Comment by MuhammadTalhaRao** [comment]
This version was not supposed to use *GetNLS* but it still is using that function. Quick fix for this is to revert to old version as i found this in 2.6.15

**Comment by sijms** [comment]
fixed in 2.6.16
